pointerevents/pointerevent_attributes_mouse-manual.html

```diff
@@ -17,68 +17,87 @@
             // will fail unless the async_test is created with the var name "test_pointerEvent".
             add_completion_callback(showPointerTypes);
 
+            function checkPointerEventAttributes(event, targetBoundingClientRect, testNamePrefix, pointerEventInterface) {
+                if (detected_eventTypes[event.type])
+                    return;
+                var pointerTestName = testNamePrefix + ' ' + event.pointerType + ' ' + event.type;
+                test(function () {
+                    assert_equals(event.pointerType, 'mouse', 'pointerType should be mouse');
+                }, pointerTestName + ".pointerType attribute is correct.");
+
+                detected_pointertypes[event.pointerType] = true;
+                // Test button and buttons
+                if (event.type == 'pointerdown') {
+                    test(function() {
+                        assert_true(event.button == 0, "If left mouse button is pressed button attribute is 0")
+                    }, pointerTestName + "'s button attribute is 0 when left mouse button is pressed.");
+                    test(function() {
+                        assert_true(event.buttons == 1, "If left mouse button is pressed buttons attribute is 1")
+                    }, pointerTestName + "'s buttons attribute is 1 when left mouse button is pressed.");
+                } else if (event.type == 'pointerup') {
+                    test(function() {
+                        assert_true(event.button == 0, "If left mouse button is just released button attribute is 0")
+                    }, pointerTestName + "'s button attribute is 0 when left mouse button is just released.");
+                    test(function() {
+                        assert_true(event.buttons == 0, "If left mouse button is just released buttons attribute is 0")
+                    }, pointerTestName + "'s buttons attribute is 0 when left mouse button is just released.");
+                } else {
+                    test(function() {
```

Comment:
The texts in the following two tests are misleading: these are not related to mouse-button release activity at all. Instead, they should simply say "pointerout's button should be x" etc.

While doing this, please consider combining all button/buttons into a single test per event type. We have 515 asserts in total, perhaps inherited ones should be combined to shorten the long list of PASSes.